src/main/java/es/unizar/smartcampuz/infrastructure/service/SmtpMailService.java

```diff
@@ -0,0 +1,65 @@
+package es.unizar.smartcampuz.infrastructure.service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+/**
+ * He creado los ficheros RootController.java, SmtpMailService.java
+ * He modificado el AdminDashboardController para enviar el email cuando aprueban o no la reserva (en /reservation)
+ * He modificado el application.properties con las credenciales de gmail y la configuración de smtp.
+ * @author catalin
+ */
```

Comment:
I don't understand the purpose of this comment.
If this is a class definition comment or a whatever final comment, It must be written in an impersonal form and translated to English. 
If this is a temporal comment to the rest of the team members, then ignore this note.

Whatever it may be, a clarification would be welcome 😄